libfwupdplugin/tests/bios-attrs/dell-xps13-9310/dell-wmi-sysman/attributes/EmbSataRaid/display_name

```diff
@@ -1 +1 @@
-SATA/NVMe Opeartion
+SATA/NVMe Operation
```

Comment:
I believe this is human readable and not tied to the underlying value, but I could be very wrong.

Reply:
@superm1 ?

Reply:
This was captured from a real machine; so it's a BIOS bug.  We shouldn't change it.